demos/demo-react-ts/src/hooks/useCti.ts

```diff
@@ -69,36 +78,85 @@ class CallingExtensionsWrapper implements CallingExtensionsContract {
     this._incomingNumber = number;
   }
 
+  get iframeLocation() {
+    return this._iframeLocation;
+  }
+
+  set iframeLocation(location: string) {
+    this._iframeLocation = location;
+  }
+
   initialized(userData: OnInitialized) {
+    if (userData.iframeLocation) {
+      this._iframeLocation = userData.iframeLocation;
+    }
+
     return this._cti.initialized(userData);
   }
 
   userAvailable() {
+    if (this._iframeLocation === "remote") {
+      this.broadcastMessage({ type: "USER_AVAILABLE" });
+      return;
+    }
+
     return this._cti.userAvailable();
   }
 
   userUnavailable() {
+    if (this._iframeLocation === "remote") {
+      this.broadcastMessage({ type: "USER_UNAVAILABLE" });
+      return;
+    }
+
     return this._cti.userUnavailable();
   }
 
   userLoggedIn() {
+    if (this._iframeLocation === "remote") {
+      this.broadcastMessage({ type: "LOGGED_IN" });
+      return;
+    }
+
     return this._cti.userLoggedIn();
   }
 
   userLoggedOut() {
+    if (this._iframeLocation === "remote") {
+      this.broadcastMessage({ type: "LOGGED_OUT" });
+      return;
+    }
+
     return this._cti.userLoggedOut();
   }
 
   incomingCall(callDetails: OnIncomingCall) {
+    // Triggered from remote
+    if (this._iframeLocation === "remote") {
+      this.broadcastMessage({ type: "INCOMING_CALL", payload: callDetails });
+      return;
+    }
+
+    // Triggered from popup
+    if (this._iframeLocation === "popup") {
+      this.broadcastMessage({ type: "INCOMING_CALL", payload: callDetails });
+    }
```

Comment:
@hemang-thakkar User can trigger an incoming call from any of the iframes that are open, whether it is in a remote or popup